### ISSUE TITLE
CancelToken property not available in custom axios instance

### ISSUE THREAD

**Issue by hsz** [simoneb/axios-hooks#80]
After latest update related to the `CancelToken`, there is an issue with:

https://github.com/simoneb/axios-hooks/blob/307c7c1565e9f2d36f0e90bd19a86815dde2d8ce/src/index.js#L148

Everything works when you use `AxiosStatic`, imported directly from `axios.
But when you pass your own instance through the options, like:

```
configure({
  axios: axios.create({
    baseURL: theme.config.api.endpoint,
  }),
});
```

it is `AxiosInstance` used which doesn't contain `CancelToken` property anymore and it ends up with the following error in mentioned line:

```
TypeError: Cannot read property 'source' of undefined
```

**Comment by arielserafini** [comment]
I've stumbled upon the same issue. Ultimately had to downgrade to 1.5.0.

**Comment by simoneb** [comment]
Fixed in 1.7.2

**Comment by hsz** [comment]
Commit looks fine, I'll test it tomorrow. Thanks!
